.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
+npm-debug.log
```

Comment:
I always leave this out - if your commit contains an npm-debug.log, you need to fix that before pushing your commit, not commit despite there being a npm failure somewhere. Seeing that file is a very strong signal that something is wrong and you should not be pushing yet.